gaia_tools/load/spec.py

```diff
@@ -0,0 +1,143 @@
+import os
+import glob
+import tqdm
+import numpy as np
+import pandas as pd
+from astropy.io import ascii
```

Comment:
Don't think you use ``pandas``? It's not in the requirements: https://github.com/jobovy/gaia_tools/blob/main/setup.py#L19

Reply:
I have removed pandas now